awsshell/utils.py

```diff
@@ -118,3 +119,16 @@ def file_contents(self, filename, binary=False):
 
     def file_exists(self, filename):
         return filename in self._file_mapping
+
+
+def force_unicode(obj, encoding='utf8'):
+    """Recursively search lists and dicts for strings to encode as unicode."""
+    if isinstance(obj, dict):
+        for key in obj:
+            obj[key] = force_unicode(obj[key])
+    elif isinstance(obj, list):
+        obj = [force_unicode(val) for val in obj]
+    elif isinstance(obj, six.string_types):
```

Comment:
It would fail the encoding, causing an exception. I'll wrap that part in a try catch so that worst case is strings that are not utf8 compatible will not be converted.